molecule/java11/playbook.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+- name: Java 11 Tests
+  hosts: all
+  roles:
+    - role: java
+      sansible_java_set_as_default: no
+      sansible_java_vendor: openjdk
+      sansible_java_version: 8
```

Comment:
Do we need to install java8 before java11?

Reply:
just a test to make sure we can have both.